support-frontend/assets/components/legal/termsPrivacy/termsPrivacy.jsx

```diff
@@ -98,14 +104,30 @@ function TermsPrivacy(props: PropTypes) {
   const isNotOneOffContribution = props.contributionType !== 'ONE_OFF';
 
   const shouldShowPhilanthropicAsk =
-    isUSContributor && isNotOneOffContribution
+    isUSContributor && isNotOneOffContribution;
+
+  const recurringCopy = () => {
+    const now = new Date();
+    const closestDayCopy = now.getDate() >= 29 ? ', or closest day thereafter' : '';
+    const amountCopy = formatAmount(
+      currencies[props.currency],
+      spokenCurrencies[props.currency],
+      parseFloat(props.amount),
+      false,
+    );
+
+    if (props.contributionType === 'MONTHLY') {
+      return `We will attempt to take payment of ${amountCopy}, on the ${getDateWithOrdinal(now)} day of every month${closestDayCopy}, from now on until you cancel your contribution. Payments may take up to 6 days to be recorded in your bank account.`;
+    }
+    return `We will attempt to take payment of ${amountCopy} on the ${getDateWithOrdinal(now)} day of ${getLongMonth(now)} every year${closestDayCopy}, from now until you cancel your contribution. Payments may take up to 6 days to be recorded in your bank account.`;
```

Comment:
Do we need the `closestDay` copy here? Because it's the same month just a year later the date will always exist